static/app/views/app/index.spec.tsx

```diff
@@ -69,6 +69,38 @@ describe('App', function () {
     user.flags.newsletter_consent_prompt = false;
   });
 
+  it('renders PartnershipAgreement', async function () {
+    ConfigStore.set('partnershipAgreementPrompt', {partnerDisplayName: 'Foo', agreements:['standard', 'partner_presence']});
+    render(
+      <App {...routerProps}>
+        <div>placeholder content</div>
+      </App>
+    );
+
+    expect(
+      await screen.findByText(/This organization is created in partnership with Foo/)
```

Comment:
i notice we're using `/` for strings in a lot of our tests.... 🤔 

This seems strange...

Why are we using regex rather than just strings?

Reply:
I was going for partial string match because this string is dynamic and has links and stuff. `findByText("This organization is created in partnership with Foo")` doesn't find it unless it's the whole label. Let me know if you have other suggestions.